firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp

```diff
@@ -313,6 +313,13 @@ void turnOnSpi(spi_device_e device) {
 				engineConfiguration->spi3MisoMode);
 #endif /* STM32_SPI_USE_SPI3 */
 	}
+	if (device == SPI_DEVICE_4) {
+#if STM32_SPI_USE_SPI4
+//		scheduleMsg(&logging, "Turning on SPI4 pins");
+		/* there is no nofiguration fields for SPI4 in engineConfiguration, rely on board init code
```

Comment:
"nofiguration"? probably "configuration"?

Shall we PR spi4 configuration pins?

Reply:
Actually no one is using SPI4. Only me. While I'm good with init code in my own board.c.